medical_prescription_state/data/base_kanban_stage.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo noupdate="1">
+
+
+    <!-- Prescription Order States -->
+
+    <record id="prescription_order_state_unverified" model="base.kanban.stage">
```

Comment:
Looks like this line is causing an error: "External ID not found in the system: medical.prescription.order" while parsing /home/travis/build/LasLabs/vertical-medical/medical_prescription_state/data/base_kanban_stage.xml:10